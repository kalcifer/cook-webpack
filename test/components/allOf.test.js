import React from 'react';

import renderer from 'react-test-renderer';
import AllOf from '../../components/allOf.js';

test('AllOf component', () => {
    const children = ['String', 'Minimum2'];
    const component = renderer.create(<AllOf children={children}/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot(); 
}) 