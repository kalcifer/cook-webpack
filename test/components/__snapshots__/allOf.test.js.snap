exports[`test AllOf component 1`] = `
<div
  data-type="allOf">
  <div>
    String
    <div>
       And 
    </div>
  </div>
  <div>
    Minimum2
  </div>
</div>
`;
